pandas/_typing.py

```diff
@@ -129,6 +129,14 @@
 DtypeArg = Union[Dtype, Dict[Hashable, Dtype]]
 DtypeObj = Union[np.dtype, "ExtensionDtype"]
 
+# converters
```

Comment:
@jreback, how about these new aliases for `converters` and `parse_dates`? (FYI: `DtypeArg` alias was first added for IO SQL #37546).

Reply:
sounds good